app.py

```diff
@@ -129,13 +131,18 @@ def discard_form(form_id):
 @auth.login_required
 @swag_from('docs/swagger/search_form.yaml')
 def search():
-    # Get JSON search prompt from the request
-    query = request.json
-    if query is None:
-        return jsonify(error="Missing search prompt!"), HTTPStatus.BAD_REQUEST
-    search_query = SearchQuery(**query)
+    # Database cursor
+    cursor = conn.cursor()
+
+    user_id = request.args.get('user_id')
+    if user_id is None:
+        return jsonify(error="Missing username!"), HTTPStatus.BAD_REQUEST
```

Comment:
Should be user not registered or something better representing the issue. We aren't using Username so far, we use email address